internal/service/ec2/ec2_ami_data_source.go

```diff
@@ -217,6 +217,10 @@ func dataSourceAMI() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"uefi_data": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Please also add the same to `internal/service/ec2/ec2_ami_copy.go` as the `aws_ami_copy` resource's schema needs the same attributes as `aws_ami` (they shared a Read function).